### ISSUE TITLE
Why Svelte....

### ISSUE THREAD

**Issue by sdudnic** [sciactive/pnotify#412]
Fist thing I expect from a documentation, a basic example using nothing at all, any additional library NPM, Angular, Bambular....

Could you just display a success message using simple html and pure js? 

What is the strict minimum to display a notification?

Is

```
https://cdn.jsdelivr.net/npm/@pnotify/core@5.2.0/dist/PNotify.min.js
https://cdn.jsdelivr.net/npm/@pnotify/core@5.2.0/BrightTheme.min.css
```

enought to display a success notification?

**Comment by hperrin** [comment]
Yes. You don't need to include Svelte. I chose Svelte as the framework to write PNotify in because it compiles down to vanilla JavaScript, so there is no need to include any libraries.
